### ISSUE TITLE
Looking for new maintainer

### ISSUE THREAD

**Issue by DrMcCoy** [DrMcCoy/tg3442_firewall#5]
**TL;DR: Somebody please take over this script, also seeking for input in proper vetting of candidates.**

I wrote this little scripts when I got the Arris TG3442 from my ISP after moving to a new flat and upgrading my internet connection to 1 Gbit/s DOCSIS 3.1, from 32 Mbit/s on an older DOCSIS standard available in my previous flat.

The other option for a router was the Fritz!Box 6591, but that wasn't yet available at the time of my move (and, in fact, got delayed a few months even after the projected release). It was finally made available to me earlier this year and I jumped at the chance, because I found the Arris TG3442 pretty miserable. And yes, I'm far happier with the Fritz!Box 6591 now.

This, unfortunately, means I don't have Arris TG3442 anymore, since I had to send it back to my ISP. So I can't develop any new features or even fixes to this script.

I'm happy to answer questions (as far as I can) and merge pull requests, but those will be untested on my end. It would probably be best if somebody else with access to an Arris TG3442 could take over maintenance of this tg3442_firewall script.

Now, I'm a bit wary of just giving this over to anybody sight-unseen, what with the recent cases of new maintainer doing shady stuff. But I'm also unsure what a proper vetting procedure would be. I guess I'm at least looking for somebody with own public code on their GitHub page, somebody I can link to public persona, somebody with established relationships to other trusted individuals. Sure, this script is nothing high-profile, but still, I want to do my due dilligence.

So, if anybody has any ideas, please step up.
